frontend/shared/components/src/events/composables/useEventPermissions.ts

```diff
@@ -0,0 +1,84 @@
+import { AccessRight, NamedObject, Organization, OrganizationTag, PermissionsResourceType } from '@stamhoofd/structures';
+import { useAuth, usePlatform } from '../../hooks';
+
+export function useEventPermissions() {
+    const auth = useAuth();
+    const platform = usePlatform();
+    const permissions = auth.permissions;
+
+    function canAdminSome() {
```

Comment:
Naamgeving mag wat leesbaarder in al deze methodes
 ```suggestion
    function canAccessSome() {
```